views/pwas/view.hbs

```diff
@@ -48,6 +48,8 @@
         <div class="pwabox-body">
           <h3>{{pwa.name}}</h4>
           <p>{{pwa.description}}</p>
+          <p>Added: {{pwa.created}}</p>
```

Comment:
How should we format the dates? Is just year, month and day enough or we also want to add hour + minute?